ApacheConnector/CMakeLists.txt

```diff
@@ -19,7 +21,14 @@ target_include_directories(mod_poco
 		${APACHE2_INCLUDE_DIRS}
 		${CMAKE_CURRENT_SOURCE_DIR}/src
 )
-target_link_libraries(mod_poco PUBLIC Poco::Util Poco::Net Apache::Apr Apache::Aprutil)
+set(DEPLIBS Poco::Util Poco::Net Apache::Apr Apache::Aprutil)
+if(WIN32)
+	list(APPEND DEPLIBS Apache::httpd)
+endif(WIN32)
+target_link_libraries(mod_poco PUBLIC ${DEPLIBS})
```

Comment:
```suggestion
target_link_libraries(mod_poco PUBLIC Poco::Util Poco::Net Apache::Apr Apache::Aprutil)
if(WIN32)
	target_link_libraries(mod_poco PUBLIC Apache::httpd)
endif(WIN32)
```
Keep it simple :-)

Reply:
I think this suggestion is wrong.
AFAIK, target_link_libraries() is **NOT** cumulative. So: your suggestion would result in missing link libraries
Poco::Util Poco::Net,  Apache::Apr and Apache::Aprutil
if building on Windows?!